### PR TITLE
Add django-elasticsearch-dsl

### PR DESCRIPTION
**Pull request by MichelML**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MichelML** [comment]
@dopplershift relates to https://github.com/conda-forge/staged-recipes/pull/14686

**Comment by MichelML** [comment]
ready to merge @dopplershift :+1: 

@conda-forge-admin, please ping team

File: recipes/django-elasticsearch-dsl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+test:
+  imports:
+    - django_elasticsearch_dsl
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sabricot/django-elasticsearch-dsl
```
Comment (dopplershift) [inline recipes/django-elasticsearch-dsl/meta.yaml:37]: This actually resolves now to https://github.com/django-es/django-elasticsearch-dsl  Can you update? I was really confused for a minute.

File: recipes/django-elasticsearch-dsl/LICENSE [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Apache Software License 2.0
+
+Copyright (c) 2016, Sabre
+
+Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (dopplershift) [inline recipes/django-elasticsearch-dsl/LICENSE:5]: Looks like the license is included in the PyPI tarball, so you should not include this file.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Found a couple more things now.

**Comment by MichelML** [comment]
Thanks @dopplershift , will do the change asap tomorrow

**Comment by MichelML** [comment]
I think it's ready now @dopplershift , thanks for the feedback! :pray:

**Review by dopplershift — APPROVED** [review]

**Pull request merged**